packages/task/src/browser/task-configuration-model.ts

```diff
@@ -18,8 +18,9 @@ import URI from '@theia/core/lib/common/uri';
 import { Emitter, Event } from '@theia/core/lib/common/event';
 import { Disposable, DisposableCollection } from '@theia/core/lib/common/disposable';
 import { TaskCustomization, TaskConfiguration, TaskConfigurationScope } from '../common/task-protocol';
-import { PreferenceProvider, PreferenceProviderDataChanges, PreferenceProviderDataChange } from '@theia/core/lib/browser';
-import { isObject } from '@theia/core/lib/common';
+import { PreferenceProviderDataChanges, PreferenceProviderDataChange, isObject } from '@theia/core/lib/common';
+import { PreferenceProvider } from '@theia/core/lib/common/preferences/preference-provider';
+import { JSONValue } from '@theia/core/shared/@lumino/coreutils';
```

Comment:
Lumino import

Reply:
Where do we have JSON types in @theia/core?